packages/ckeditor5-widget/src/widget.ts

```diff
@@ -649,13 +651,16 @@ function findClosestEditableOrWidgetAncestor( element: ViewElement ): ViewElemen
  */
 function getElementFromMouseEvent( view: EditingView, domEventData: DomEventData<MouseEvent> ): ViewElement | null {
 	const domRange = getRangeFromMouseEvent( domEventData.domEvent );
-
-	if ( !domRange ) {
-		return null;
+	let viewRange: ViewRange | null = null;
+
+	if ( domRange ) {
+		viewRange = view.domConverter.domRangeToView( domRange );
+	} else {
```

Comment:
Maybe we should verify it the `target` is here as this is a view node so it could have no mapping from DOM:

```suggestion
	} else if ( domEventData.target ) {
```

The `target` in the event data is marked as it always returns an element but it could be a mistake.